elastic/integrations
# Update system test cases for galera_status

- Enhancement


## What does this PR do?

This PR adds system test cases for galera_status datastream for mysql.

## Checklist




## How to test this PR locally

Add mysql integration, setup and enable galera_status datastream and metrics should come.

## Related issues



- Closes #4313
